keel-core/src/test/kotlin/com/netflix/spinnaker/keel/actuation/ResourceActuatorTests.kt

```diff
@@ -93,18 +93,13 @@ internal class ResourceActuatorTests : JUnit5Minutests {
       every { plugin2.supportedKind } returns SupportedKind(parseKind("plugin2/bar@v1"), DummyArtifactVersionedResourceSpec::class.java)
     }
 
-    after {
-      resourceRepository.dropAll()
```

Comment:
Just to make sure I understand, the reason we don't need a `clearAllMocks()` to replace this here is that the mocks are initialized in the fixture and the fixture is newly instantiated in every test?

Reply:
Yes. I'm always suspicious of any use of `clearAllMocks` as it suggests that something is leaking.